source/material_model/viscoelastic.cc

```diff
@@ -252,12 +255,17 @@ namespace aspect
                                                                         average_elastic_shear_modulus,
                                                                         dte);
 
-          // Fill elastic outputs if they exist
+          // Fill elastic additional outputs if they exist
```

Comment:
As we have ElasticOutputs and ElasticAdditionalOutputs (which is a bit confusing) and the code in the lines below makes it clear which of those is filled, I would change this comment to something like "Fill the material properties that are part of the elastic additional outputs (in this case the shear modulus)".